tools/rpkg/tests/testthat/test_tbl__duckdb_connection.R

```diff
@@ -0,0 +1,68 @@
+skip_on_cran()
+skip_if_no_R4 <- function() {
+  if (R.Version()$major < 4) {
+    skip("R 4.0.0 or newer not available for testing")
+  }
+}
+
+test_that("Parquet files can be registered with dplyr::tbl()", {
+  con <- DBI::dbConnect(duckdb::duckdb())
+  on.exit(DBI::dbDisconnect(con, shutdown = TRUE))
+
+  tab0 <- dplyr::tbl(con, "data/userdata1.parquet")
+  expect_true(inherits(tab0, "tbl_duckdb_connection"))
+  expect_true(tab0 |> dplyr::count() |> dplyr::collect() == 1000)
```

Comment:
Since native pipe `|>` can only be used after R4.1.0, don't we need to rewrite the test to not use native pipes or skip R 4.0.x?

Reply:
Fix in  #3318